Gems/AtomLyIntegration/AtomImGuiTools/Code/Source/AtomImGuiToolsSystemComponent.cpp

```diff
@@ -111,6 +111,11 @@ namespace AtomImGuiTools
     {
         if (ImGui::BeginMenu("Atom Tools"))
         {
+            if (ImGui::MenuItem("Dump loaded Asset info", ""))
```

Comment:
is it possible to hint somehow that the destination is in the user/log folder?  is the second parameter to MenuItem for some help text?

Reply:
I can add trace info about the path so it would be printed out in debug console.